### PR TITLE
Fix Timespan Remarks

### LINKED ISSUE #9058: TimeSpan.TotalMicroseconds and TimeSpan.TotalNanoseconds (NET7+) Remarks section displayed incorrectly
TimeSpan.TotalMicroseconds and TimeSpan.TotalNanoseconds (NET7+) Remarks section displayed incorrectly

https://learn.microsoft.com/en-us/dotnet/api/system.timespan.totalnanoseconds?view=net-7.0
https://learn.microsoft.com/en-us/dotnet/api/system.timespan.totalmicroseconds?view=net-7.0

### PR DESCRIPTION
**Pull request by tarekgh**
Fixes #9058

### CODE REVIEW

**Comment by tarekgh** [comment]
@gewarren is there a way I can see the visuals of this change?

File: xml/System/TimeSpan.xml [hunk 1]
```diff
@@ -5139,9 +5139,9 @@ The value of this constant is 10.
 
 This property converts the value of this instance from ticks to microseconds.
              This number might include whole and fractional microseconds.
-            
-             The <xref:System.TimeSpan.TotalMicroseconds> property represents whole and fractional microseconds,
-             whereas the <xref:System.TimeSpan.Microseconds> property represents whole microseconds.
+
+The <xref:System.TimeSpan.TotalMicroseconds%2A> property represents whole and fractional microseconds,
```
Comment (gewarren) [inline xml/System/TimeSpan.xml:5143]: `%2A` is only needed if you want to link to all overloads of a method. It's not needed for properties.

**Comment by gewarren** [comment]
> @gewarren is there a way I can see the visuals of this change?

Yes you can see it here: https://review.learn.microsoft.com/en-us/dotnet/api/system.timespan.totalmicroseconds?view=net-8.0&branch=pr-en-us-9133#remarks

File: xml/System/TimeSpan.xml (hunk 1 above)
Comment (gewarren) [reply xml/System/TimeSpan.xml:5143]: Yes, if the text is indented by 4 spaces in Markdown, it's automatically converted into a code block.

**Review by gewarren — APPROVED** [review]

**Pull request merged**